doc/content/source/vectorpostprocessors/Fragility.md

```diff
@@ -12,6 +12,22 @@ demand distribution and capacity distribution and calculate the conditional prob
 in each bin, and 3) fit a lognormal distribution in the conditional probabilities to calculate a
 fragility. This kind of fragility is also referred to as 'enhanced fragility'.
 
+Fitting a lognormal fragility function given analysis results requires optimization to find
```

Comment:
Please change this to:
Fitting a lognormal fragility function to the probabilities of failure requires an optimization algorithm to find the parameters, median and lognormal standard deviation. Two methods are currently available for calculating these fragility parameters: 'brute force' and 'Stochastic Gradient Descent (SGD)'.